playbooks/roles/minishift/tasks/main.yml

```diff
@@ -74,9 +79,19 @@
   when: ((minishift_profile_status.stdout_lines and minishift_profile_status.stdout_lines[0]|lower == "does not exist") or
         (minishift_profile_status.rc != 0)) and (minishift_external_vm_ip == "")
 
-# Start minishift on an external VM if configured
+# Start minishift on an external VM if configured, then shut it down (if it was configured not to start, but we wanted the profile to exist)
```

Comment:
Looking into this a bit more, the `minishift profile set` command on [line 25](https://github.com/CentOS-PaaS-SIG/contra-env-setup/pull/175/files#diff-0d002d82212b0a8c719d5df0c6de01c396bbfd48de83b2168ad060c64632106aR25) should already create the profile without the need to start the minishift machine.

Reply:
Oh interesting, so I suppose I could change it from
- Starting up minishift
- Stopping if `start_minishift == false`
to
- Starting up minishift if `start_minishift == true`?

Reply:
Yes, I think that should be it,

Reply:
Okay, I changed the logic